build/openwisp_dashboard/urls.py

```diff
@@ -16,7 +15,5 @@
     url(r'^', include('openwisp_radius.urls', namespace='freeradius')),
 ]
 
-urlpatterns += openwisp_controller_urls()
-urlpatterns += openwisp_topology_urls()
```

Comment:
great, are topology URLs served somewhere else right?

One quesiton, have you checked what happens when trying to visualize the topology graph in the admin?

Reply:
@nemesisdesign 

Thanks for pointing this out, no it's not working, I am getting an error : `django.urls.exceptions.NoReverseMatch: Reverse for 'receive_topology' not found. 'receive_topology' is not a valid view function or pattern name.`

Either it needs to be added in admin namespace or topology `api` needs to stick with dashboard! :smile:

Reply:
I think that we need to start to make the modules configurable about the base URL of the APIs that are called in the admin, there are quite several instances of this.

Alternatively for the moment yes we can also add the same views into the admin namespace if that does not generate too much duplication.